runtime/phala/src/lib.rs

```diff
@@ -0,0 +1,1214 @@
+>;
+
+construct_runtime! {
+    pub enum Runtime where
+        Block = Block,
+        NodeBlock = opaque::Block,
+        UncheckedExtrinsic = UncheckedExtrinsic,
+    {
+        // System support stuff
+        System: frame_system::{Pallet, Call, Config, Storage, Event<T>} = 0,
+        Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 1,
+        RandomnessCollectiveFlip: pallet_randomness_collective_flip::{Pallet, Storage} = 2,
+        Utility: pallet_utility::{Pallet, Call, Event} = 3,
+        Multisig: pallet_multisig::{Pallet, Call, Storage, Event<T>} = 4,
+        Proxy: pallet_proxy::{Pallet, Call, Storage, Event<T>} = 5,
+        Vesting: pallet_vesting::{Pallet, Call, Storage, Event<T>, Config<T>} = 6,
+        Scheduler: pallet_scheduler::{Pallet, Call, Storage, Event<T>} = 7,
+
+        // Parachain staff
+        ParachainInfo: pallet_parachain_info::{Pallet, Storage, Config} = 20,
+        ParachainSystem: cumulus_pallet_parachain_system::{Pallet, Call, Config, Storage, Inherent, Event<T>, ValidateUnsigned} = 21,
+
+        // Monetary stuff
+        Balances: pallet_balances::{Pallet, Call, Storage, Config<T>, Event<T>} = 40,
+        TransactionPayment: pallet_transaction_payment::{Pallet, Storage} = 41,
+        Assets: pallet_assets::{Pallet, Call, Storage, Event<T>} = 42,
+
+        // Collator support. the order of these 5 are important and shall not change.
+        Authorship: pallet_authorship::{Pallet, Call, Storage} = 50,
+        CollatorSelection: pallet_collator_selection::{Pallet, Call, Storage, Event<T>, Config<T>} = 51,
+        Session: pallet_session::{Pallet, Call, Storage, Event, Config<T>} = 52,
+        Aura: pallet_aura::{Pallet, Storage, Config<T>} = 53,
+        AuraExt: cumulus_pallet_aura_ext::{Pallet, Storage, Config} = 54,
+
+        // Governance
+        Identity: pallet_identity::{Pallet, Call, Storage, Event<T>} = 60,
+        Democracy: pallet_democracy::{Pallet, Call, Storage, Config<T>, Event<T>} = 61,
+        Council: pallet_collective::<Instance1>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 62,
+        Treasury: pallet_treasury::{Pallet, Call, Storage, Config, Event<T>} = 63,
+        Bounties: pallet_bounties::{Pallet, Call, Storage, Event<T>} = 64,
+        Lottery: pallet_lottery::{Pallet, Call, Storage, Event<T>} = 65,
+        TechnicalCommittee: pallet_collective::<Instance2>::{Pallet, Call, Storage, Origin<T>, Event<T>, Config<T>} = 66,
+        TechnicalMembership: pallet_membership::<Instance1>::{Pallet, Call, Storage, Event<T>, Config<T>} = 67,
+        PhragmenElection: pallet_elections_phragmen::{Pallet, Call, Storage, Event<T>, Config<T>} = 68,
+        Tips: pallet_tips::{Pallet, Call, Storage, Event<T>} = 69,
+
+        // Main, starts from 80
+
+        // TODO: Remove sudo in the future
+        Sudo: pallet_sudo::{Pallet, Call, Storage, Config<T>, Event<T>} = 255,
```

Comment:
Not sure if 255 is supported

Reply:
learned from Acala <https://github.com/AcalaNetwork/Acala/blob/master/runtime/acala/src/lib.rs#L1833>